examples/submenus.py

```diff
@@ -0,0 +1,90 @@
+"""
+Create a CLI with a nested command structure as follows. The commands 'second' and 'third' navigate the CLI to the scope
+of the submenu. Nesting of the submenus is done with the cmd2.AddSubmenu() decorator.
+
+      (Top Level)----second----->(2nd Level)----third----->(3rd Level)
```

Comment:
Thank you for creating a nice example to go along with the submenu feature.  It goes a long way to explaining how to properly use it.